src/main/java/TaskList.java

```diff
@@ -0,0 +1,76 @@
+import java.util.ArrayList;
+
+/**
+ * Manages an ArrayList of Tasks.
+ */
+public class TaskList {
+
+    private ArrayList<Task> taskList;
+
+    public TaskList() {
+        this.taskList = new ArrayList<Task>();
+    }
+
+    /**
+     * Adds a new string as ToDo to the ArrayList and prints the Task added.
+     *
+     * @param taskName
+     */
+    public void addToDo(String taskName) {
+        Task newToDo = new ToDo(taskName);
+        taskList.add(newToDo);
+        System.out.println("Added:" + System.lineSeparator() + "   " + newToDo);
+    }
+
+    /**
+     * Adds a new string as Deadline to the ArrayList and prints the Task added.
+     *
+     * @param taskName
+     * @param deadline
+     */
+    public void addDeadline(String taskName, String deadline) {
+        Task newDeadline = new Deadline(taskName, deadline);
+        taskList.add(newDeadline);
+        System.out.println("Added:" + System.lineSeparator() + "   " + newDeadline);
+    }
+
+    /**
+     * Adds a new string as Event to the ArrayList and prints the Task added.
+     *
+     * @param taskName
+     * @param eventTime
+     */
+    public void addEvent(String taskName, String eventTime) {
+        Task newEvent = new Event(taskName, eventTime);
+        taskList.add(newEvent);
+        System.out.println("Added:" + System.lineSeparator() + "   " + newEvent);
+    }
+
+    /**
+     * Prints out the list of Tasks.
+     */
+    public void displayTasks() {
+        System.out.println("Total " + taskList.size() + " tasks to be completed:");
+        for (int i = 0; i < taskList.size(); i++) {
+            System.out.println((i + 1) + ". " + taskList.get(i));
```

Comment:
I really like the effort to make these 2 lines of code readable. Nice use of appropriate whitespace and parenthesis!